packages/twenty-e2e-testing/tests/workspaces.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { test, expect } from '@playwright/test';
+import { sh } from '../drivers/shell_driver';
+
+const date = new Date();
+
+test.afterEach(async ({ page, browserName }) => {
+  await page.screenshot({
+    path: `./packages/twenty-e2e-testing/results/screenshots/${browserName}/${date.toISOString()}.png`,
+  });
+});
+
+test.describe('', () => {
+  test('Testing logging', async ({ page }) => {
+    await page.goto('/');
+    await page.getByRole('button', { name: 'Continue With Email' }).click();
+    await page.getByPlaceholder('Email').fill('tim@apple.dev');
+    await page.getByRole('button', { name: 'Continue' }).click();
```

Comment:
The test is failing here for me, seems like it cannot get the Continue button.

Reply:
That's weird, is it because the button is not active as the Email field wasn't triggered by the click? Please use `yarn test:e2e:ui workspaces` (or `yarn playwright test --debug workspaces`) to see where it fails

Reply:
Okay, that's because of optimizeDeps line, since there's blank page, test can't find button and can't log in.